<!DOCTYPE html>
<html lang="" xml:lang="">
  <head>
    <title>Thesis Presentation</title>
    <meta charset="utf-8" />
    <script src="libs/header-attrs-2.7/header-attrs.js"></script>
    <link href="libs/remark-css-0.0.1/default.css" rel="stylesheet" />
    <link href="libs/remark-css-0.0.1/metropolis.css" rel="stylesheet" />
    <link href="libs/remark-css-0.0.1/metropolis-fonts.css" rel="stylesheet" />
    <link href="libs/panelset-0.2.6/panelset.css" rel="stylesheet" />
    <script src="libs/panelset-0.2.6/panelset.js"></script>
  </head>
  <body>
    <textarea id="source">

class: primary

&lt;style type="text/css"&gt;

p.caption {
  font-size: 0.6em;
}

.large { font-size: 200% }

.medium-large { font-size: 130% }

.small{ font-size: 75% }

.cv{ font-size: 60% }

.tiny{ font-size: 40% }

.center-left {
  position:          relative;
  top:               50%;
  transform:         translateY(50%);
}
.center-right {
  position:          relative;
  top:               50%;
  transform:         translateY(10%);
}

.remark-slide-content {
  background-color: #FFFFFF;
  border-top: 80px solid #662e6b;
  font-size: 28px;
  font-weight: 300;
  line-height: 1.5;
  padding: .5em 3em .5em 3em
}

.inverse {
  background-color: #662e6b;
  text-shadow: none;
}

.right-column {
	color: #000000;
	width: 30%;
	height: 92%;
	float: right;
}

.left-column {
  color: #000000;
  width: 68%;
  float: left;
}

.remark-slide-number {
	display: none;
}

.remark-slide-content:after {
    content: "";
    position: absolute;
    bottom: 0px;
	left: 20px;
    height: 60px;
    width: 400px;
	font-size: 12px;
    background-repeat: no-repeat;
    background-size: contain;
	background-image: url("img/The-Ohio-State-University-Wexner-Medical-Center.png")
}

mark {
  background-color: teal;
  color: white;
}

.my-logo-null {
content: "";
    position: absolute;
    top: 15px;
    right:   8px;
    height: 200px;
    width: 250px;
    background-repeat: no-repeat;
    background-size: contain;
    background-image: url("img/Null_model_badge.png");
}

.my-logo-KP {
content: "";
    position: absolute;
    top: 15px;
    right:   8px;
    height: 200px;
    width: 300px;
    background-repeat: no-repeat;
    background-size: contain;
    background-image: url("img/Known_positive_badge.png");
}

.my-logo-SP {
content: "";
    position: absolute;
    top: 15px;
    right:   8px;
    height: 200px;
    width: 250px;
    background-repeat: no-repeat;
    background-size: contain;
    background-image: url("img/Sample_partition_badge.png");
}


&lt;/style&gt; 

.pull-left[
&lt;h2 style=font-size:35px&gt; Integrative and Network-Based Approaches for Functional Interpretation of Metabolomic Data&lt;/style&gt;
&lt;h6 style=font-size:20px&gt;
&lt;br&gt;
Andrew Patt, Doctoral Candidate
&lt;br&gt;
&lt;br&gt;
National Center for Advancing Translational Science/The Ohio State University
&lt;/style&gt;
]

.pull-right[
&lt;img src="img/Thesis_schematic.png" width="500px" /&gt;
]





---

# C.V.

.cv[
**B.Sc. Biochemistry, Biomathematics minor (SUNY Geneseo, 2015)**

#### Published

1. RaMP: A Comprehensive Relational Database of Metabolomics Pathways
  for Pathway Enrichment Analysis of Genes and Metabolites. 2018,
  Metabolites (*Second author*)
2. Integration of Metabolomics and Transcriptomics to Identify
  Gene-Metabolite Relationships Specific to Phenotype. 2019, Methods
  Mol Biol (*First author*)
3. Challenges in proteogenomics: a comparison of analysis methods with
  the case study of the DREAM proteogenomics sub-challenge. 2019, BMC
  Bioinformatics (*Co-first author*)
4. Metabolomics and Multi-Omics Integration: A Survey of Computational
  Methods and Resources. 2020, Metabolites (*Second author*)
5. MDM2-Dependent Rewiring of Metabolomic and Lipidomic Profiles in
  Dedifferentiated Liposarcoma Models. 2020, MDPI Cancers (*Co-first author*)

#### Preprint/ In review
1. An OpenData portal to share COVID-19 drug repurposing data in real
  time. 2020, bioRxiv (*Middle author*)
2. Lipidomics and Redox Lipidomics Indicate Early-Stage Alcohol-Induced
  Liver Damage. 2021, Hepatology (*In review, Middle author*)

#### In progress
1. MetaboSPAN: Network-Based Approach for Functional Interpretation
  of Metabolomic Data
2. RaMP 2.0: update to the RaMP pathway database
]


---

# C.V.

.cv[
**B.Sc. Biochemistry, Biomathematics minor (SUNY Geneseo, 2015)**

#### Published

1. &lt;mark&gt;&lt;b&gt;RaMP: A Comprehensive Relational Database of Metabolomics Pathways
  for Pathway Enrichment Analysis of Genes and Metabolites. 2018,
  Metabolites (&lt;i&gt;Second author&lt;/i&gt;)&lt;/b&gt;&lt;/mark&gt;
2. &lt;mark&gt;&lt;b&gt;Integration of Metabolomics and Transcriptomics to Identify
  Gene-Metabolite Relationships Specific to Phenotype. 2019, Methods
  Mol Biol (&lt;i&gt;First author&lt;/i&gt;)&lt;/b&gt;&lt;/mark&gt;
3. Challenges in proteogenomics: a comparison of analysis methods with
  the case study of the DREAM proteogenomics sub-challenge. 2019, BMC
  Bioinformatics (*Co-first author*)
4. &lt;mark&gt;&lt;b&gt;Metabolomics and Multi-Omics Integration: A Survey of Computational
  Methods and Resources. 2020, Metabolites (&lt;i&gt;Second author&lt;/i&gt;)&lt;/b&gt;&lt;/mark&gt;
5. &lt;mark&gt;&lt;b&gt;MDM2-Dependent Rewiring of Metabolomic and Lipidomic Profiles in
  Dedifferentiated Liposarcoma Models. 2020, MDPI Cancers (&lt;i&gt;Co-first author&lt;/i&gt;)&lt;/b&gt;&lt;/mark&gt;

#### Preprint/ In review
1. An OpenData portal to share COVID-19 drug repurposing data in real
  time. 2020, bioRxiv (*Middle author*)
2. Lipidomics and Redox Lipidomics Indicate Early-Stage Alcohol-Induced
  Liver Damage. 2021, Hepatology (*In review, Middle author*)

#### In progress
1. &lt;mark&gt;&lt;b&gt;MetaboSPAN: Network-Based Approach for Functional Interpretation
  of Metabolomic Data&lt;/b&gt;&lt;/mark&gt;
2. &lt;mark&gt;&lt;b&gt;RaMP 2.0: update to the RaMP pathway database&lt;/b&gt;&lt;/mark&gt;
]

---
# Metabolomic pathway analysis

&lt;img src="img/Metabolomics_overview.png" width="1200px" /&gt;

---

# Biological pathway databases

.pull-left[
- Example: **Glycolysis** is a chain of 10 enzyme-catalyzed reactions
  where glucose is converted to pyruvate .tiny[(https://www.thoughtco.com/steps-of-glycolysis-373394)
  ]]
  
.pull-right[
.center[
&lt;img src="img/Glycolysis.jpeg" width="300px" /&gt;
]]

.pull-left[
- Stored in databases as lists of related analytes, or networks where nodes are metabolites and edges are reactions
]

.pull-right[
.tiny[
| Name            | # metabolites | Other features                                    |
|----------------|--------------|--------------------------------------------------|
| HMDB/SMPDB      | &gt;114,000      | Reactions, chemical class, spectra, ontologies    |
| KEGG            | &gt;18,000       | Reactions, Pathway hierarchy (BRITE)              |
| Reactome        | 1,856         | Interactive "Google Maps" pathway explorer        |
| Wikipathways    | 3,133         | Community-driven                                  |
| MetaCyc         | 16,313        | Highly curated                                    |
| PathBank        | 78,488        | Highly recent, contains many novel lipid pathways |
]
]

---

# Relational database of Metabolic Pathways (RaMP)
.pull-left[
- Multiomic pathway MySQL database integrating
information from **KEGG**, **HMDB**, **WikiPathways** and **Reactome** 
- Currently contains pathway information for &gt; 13,000 metabolites and &gt;14,000 
  transcripts, totalling 51,526 pathways and 536,245 associations
- Additional information includes chemical structures, chemical
classes, reactions, and ontologies
]

.pull-right[
&lt;img src="img/RaMP_contents.png" width="600px" /&gt;
]
---

# Using RaMP 2.0

.left-column[
.center[
&lt;img src="img/Ramp2_1.PNG" width="500px" /&gt;

&lt;img src="img/Ramp2_3.png" width="500px" /&gt;
]
]

.right-column[
- Example data to help users format input
- Tabs for each step of pathway analysis
- Pane showing underlying R functions
- Informative plots showing pathway overlap clustering results
]

---

# Pathway overlap

.pull-left[
&lt;div class="figure"&gt;
&lt;img src="img/RaMP_pathway_overlap3.svg" alt="Overlap in metabolites making up pathways in RaMP. Rows and columns are pathways, cells are Jaccard index of constituent metabolites" width="450px" /&gt;
&lt;p class="caption"&gt;Overlap in metabolites making up pathways in RaMP. Rows and columns are pathways, cells are Jaccard index of constituent metabolites&lt;/p&gt;
&lt;/div&gt;
]

.pull-right[
- Pathways often share sub-processes, resulting in functional redundancy
- This causes collinearity in pathway analysis results, making interpretation difficult
- Heatmap shows high overlap in pathway constituent metabolites,
particularly within the same databases ]

---

# Pathway overlap clustering

.center[
&lt;img src="img/Pathway_clustering.svg" width="1100px" /&gt;
]

---

# Other issues with pathway analysis

.pull-left[

![](index_files/figure-html/unnamed-chunk-10-1.png)&lt;!-- --&gt;
]

.pull-right[
- Lack of pathway annotations for metabolites (see left)
  + .small[Testing for enrichment of other annotations with better coverage can increase the number of metabolites we can incorporate into analysis]
- Uncertainty in metabolite identification
  + .small[Need to develop metabolomics-specific methodology that accounts for this uncertainty]
]

---

# MetaboSPAN

.center[
&lt;img src="img/MetaboSPAN_Schematic.png" width="1000px" /&gt;
]

---

# Validating MetaboSPAN

.center[
&lt;img src="img/Validation_schematic.png" width="900px" /&gt;
]

---

# Pathway analysis methods to compare against
&lt;/br&gt;

.small[
| Class                        | Pros  | Cons | Representative      |
|------------------------------|---|---------|---------------------|
| Over-representation analysis | Simple to implement, easy to interpret  |Depends on choice of background, ignores pathway overlap         | Fisher's exact test |
| Functional class scoring     | Self-contained hypothesis (no background)  | Hypothesis interpretation difficult, more challenging to implement | Globaltest          |
| Topological analysis         | Leverages graph theory and known pathway topology to improve analysis  | Graph-level annotations are not always available, challenging to implement        | NetGSA              |
| FELLA                        | Can enrich for reactions and pathways  | Only applicable to KEGG annotations   | FELLA               |
]

---


&lt;div class="my-logo-null"&gt;&lt;/div&gt; 

# Null model experiment
.pull-left[
.small[
| Background | Test           | .cv[Median sig. pathway count] |
|------------|----------------|--------------------------|
| Custom     | Fishers        | 1                        |
| Custom     | MetaboSPAN 10% | 2                        |
| Custom     | MetaboSPAN 15% | 3                        |
| Custom     | MetaboSPAN 20% | 4                        |
| KEGG       | Fishers        | 33                       |
| KEGG       | MetaboSPAN 10% | 33                       |
| KEGG     | MetaboSPAN 15% | 34                       |
| KEGG     | MetaboSPAN 20% | 35                         |
| N/a        | NetGSA         | 9                        |
| N/a        | MSEA           | 36                       |
| N/a        | FELLA          | 54                       |
]
]

.pull-right[
![](index_files/figure-html/unnamed-chunk-13-1.png)&lt;!-- --&gt;
]

---

&lt;div class="my-logo-KP"&gt;&lt;/div&gt; 

# Recovery of known altered pathways

.left-column[
&lt;img src="img/Known_positive_exp1.svg" width="1100px" /&gt;
]

.right-column[
.small[
- Tested nine different formulas for known positive pathways lists in terms of coverage and size

- Examined results using raw and Holm-adjusted p value

- In three scenarios post Holm adjustment, MetaboSPAN improved median recall rate

- In three scenarios using raw pval, MetaboSPAN decreased median false positive rate

]]

---

&lt;div class="my-logo-KP"&gt;&lt;/div&gt; 

# Recovery of known altered pathways

&lt;img src="img/Known_positivefigure.png" width="900px" /&gt;

---

&lt;div class="my-logo-SP"&gt;&lt;/div&gt; 

# Impact of sample size on performance

.left-column[
.small[
| Background | Test           | TPs | Median % recall | Median FPs |
|------------|----------------|---------------------------|---------------------------------------|-----------------------------------------|
| KEGG       | MetaboSPAN 20% | 5                         | 100%                                  | 3                                       |
| KEGG       | MetaboSPAN 10% | 10                        | 100%                                  | 2                                       |
| KEGG       | Fisher's       | 14                        | 93%                                  | 1                                       |
| KEGG       | MetaboSPAN 15% | 6                         | 77%                                   | 1                                       |
| Custom     | MetaboSPAN 20% | 5                         | 100%                                  | 0.5                                     |
| Custom     | MetaboSPAN 10% | 4                         | 100%                                  | 2                                       |
| Custom     | Fisher's       | 2                         | 100%                                  | 1                                       |
| Custom     | MetaboSPAN 15% | 6                         | 83%                                   | 1                                       |
| N/a        | MSEA           | 36                        | 97%                                    | 0                                       |
| N/a        | FELLA          | 14                        | 77%                                    | 1                                        |
| N/a        | NetGSA         | 24                        | 75%                                    | 2                                       |
]
]
.right-column[
&lt;/br&gt;
.small[
- TPs are true positives (significant pathways found in unpartitioned data)
- FPs are false positives (significant pathways found in partitions not found in full data)
-  MetaboSPAN was able to improve recall in KEGG background use case
and lower false positives in custom case 
]
]

---

# Real data: Similar findings in separate cohorts

.left-column[
.small[
| Background | Test           | Cohort 1 | Cohort 2 | **Overlap** |
|------------|----------------|----------|----------|---------|
| KEGG       | MetaboSPAN 20% | 10       | 31       | **.6**  |
| KEGG       | MetaboSPAN 15% | 13       | 27       | **.54** |
| KEGG       | MetaboSPAN 10% | 10       | 28       | **.5**  |
| KEGG       | Fisher's       | 14       | 31       | **.5**  |
| Custom     | MetaboSPAN 20% | 5        | 7        | **.2**  |
| Custom     | MetaboSPAN 15% | 6        | 5        | **.2**  |
| Custom     | MetaboSPAN 10% | 4        | 5        | **0**   |
| Custom     | Fisher's       | 2        | 4        | **0**   |
| N/a        | MSEA           | 36       | 46       | **.69** |
| N/a        | NetGSA         | 24       | 30       | **.46** |
| N/a        | FELLA          | 14       | 19       | **.14**     |

]
]

.right-column[
.small[
- Significant pathways by each method were found in two datasets
  generated from same cohort
  
- Only one metabolite was found in common between both datasets

- MetaboSPAN improved the concordance of Fisher's results in both
background cases ]
]

---

# Pathway anaysis conclusions

### 1. MetaboSPAN can improve the reproducibility and sensitivity of Fisher's results, with a cost to specificity

### 2. When considering an algorithm, check how many significant pathways it finds in random data

### 3. When using Fishers, be aware of the tradeoffs and assumptions that come with each background selection

### 4. Consider using an ensemble approach to identify the most trustworthy altered pathways

---

# Dedifferentiated Liposarcoma 

.pull-left[
&lt;img src="img/DDLPS_prognosis.png" width="350px" /&gt;

.tiny[
Bill KLJ, Seligson ND, Hays JL, et al. Degree of Amplification Affects Clinical Outcomes in Dedifferentiated Liposarcoma. Oncologist. 2019;24(7):989-996.
]
]

.pull-right[
- 20% of all soft tissue sarcoma diagnoses

- Prognosis depends on copy number of the MDM2 oncogene

- Some drugs (Dactinomycin, Doxorubicin) are used to treat
  liposarcoma, but they have unpredictable efficacy and toxicity.
  
]

---

# Targeting the p53-cholesterol synthesis axis

.pull-left[
- Statins (mevalonate pathway inhibitors) have been associated with
  reduced mortality in mutant p53 cancers
- MDM2 modulates p53 expression as an antagonist
- p53 downregulates transcriptional regulator of mevalonate pathway
- p53-null cell lines have been shown to withstand sterol deprived
  conditions better than wild-type

]

.pull-right[

&lt;img src="img/Atorvastatin_response.svg" width="800px" /&gt;

]

---

# Metabolomic Methods

.center[
&lt;img src="img/lipid_methods.png" width="1000px" /&gt;
]

---

# Impact of statin on DDLPS cell lipidome

.panelset[
.panel[.panel-name[MDM2 lower cells]

.pull-left[
![](index_files/figure-html/unnamed-chunk-19-1.png)&lt;!-- --&gt;
]

.pull-right[

- MDM2 low tumors have better prognosis
- MDM2 low cells responded to atorvastatin treatment
- 11 ceramides and 3 hexosylated ceramides were upregulated by atorvastatin treatment in MDM2 low cells

]
]

.panel[.panel-name[MDM2 higher cells]

.pull-left[
![](index_files/figure-html/unnamed-chunk-20-1.png)&lt;!-- --&gt;
]

.pull-right[
- MDM2 high tumors have worse prognosis
- MDM2 high cell lines did not respond to atorvastatin treatment
- Analyzing the lipidome of MDM2 high cells pre and post atorvastatin
treatment revealed only 4 lipids perturbed by treatment ]

]

]

---

# Sphingolipid metabolism as a therapeutic target

.center[
&lt;img src="img/sphingolipid_therapy.png" width="900px" /&gt;
]

---

# MDM2 status impact on ceramide levels

.panelset[
.panel[.panel-name[Metabolite panel]
.center[
![](index_files/figure-html/unnamed-chunk-22-1.png)&lt;!-- --&gt;

]
]


.panel[.panel-name[Lipid panel]
.pull-left[
![](index_files/figure-html/unnamed-chunk-23-1.png)&lt;!-- --&gt;
]

.pull-right[
&lt;/br&gt;
![](index_files/figure-html/unnamed-chunk-24-1.png)&lt;!-- --&gt;

]
]
]

---

# MetaboSPAN Results in Liposarcoma

&lt;img src="img/Network_figure_2.png" width="1000px" /&gt;

---

# In Conclusion

### 1. We characterized the metabolome/lipidome of a DDLPS model and identified sphingolipid metabolism as a potential therapeutic target

### 2. We developed RaMP, a comprehensive metabolomics functional annotations resource that harmonizes information from many large sources, and provides a convenient user interface, R package and MySQL database for ease of use

### 3. We developed and validated MetaboSPAN, a novel network-based pathway analysis algorithm that can compensate for uncertainty in identification and incomplete metabolome coverage

---

# Acknowledgements

.pull-left[
.small[
**Mathé Lab**
- Dr. Ewy Mathé, OSU/NCATS
- Tara Eicher, OSU/NCATS
- Kevin Ying, OSU
- Kyle Spencer, OSU/NCATS
- Dr. Garrett Kinnebrew, OSU


**Collaborators/Advisors**
- Dr. James Chen, OSU
- Dr. Kevin Coombes, OSU
- Dr. Zachary Abrams, OSU
- Dr. Lang Li, OSU
- Dr. Rachel Kopec, OSU

]]

.pull-right[
.small[
**Collaborators/Advisors**
- Dr. Tim Garrett, Florida State University
- Dr. Jeremy Koelmel Florida State University
- John Braisted, NCATS
- Jorge Neyra, NCATS
- NCATS informatics core

**Funding from**
- The OSU Clinical and Translational Research Informatics Training
Program (4T15LM011270-05)
- Systems and Integrative Biology training program (T32GM068412) ]]

---

# Questions?

.pull-left[
&lt;iframe src='https://gfycat.com/ifr/LongLateAcouchi' frameborder='0' scrolling='no' allowfullscreen width='480' height='390'&gt;&lt;/iframe&gt;&lt;p&gt;&lt;a href="https://gfycat.com/discover/question-gifs"&gt;from Question GIFs&lt;/a&gt; &lt;a href="https://gfycat.com/longlateacouchi-question-mark-confused-what-huh"&gt;via Gfycat&lt;/a&gt;&lt;/p&gt;
]

.pull-right[
- Slides link: https://andyptt21.github.io/Thesis_presentation_6_29_21

- RaMP link: https://rampdb.ncats.io/
]

---

# Known Positive Table

&lt;img src="img/Known_positive_table.png" width="1000px" /&gt;
    </textarea>
<style data-target="print-only">@media screen {.remark-slide-container{display:block;}.remark-slide-scaler{box-shadow:none;}}</style>
<script src="https://remarkjs.com/downloads/remark-latest.min.js"></script>
<script>var slideshow = remark.create({
"highlightStyle": "github",
"countIncrementalSlides": false,
"seal": false,
"ratio": "16:9"
});
if (window.HTMLWidgets) slideshow.on('afterShowSlide', function (slide) {
  window.dispatchEvent(new Event('resize'));
});
(function(d) {
  var s = d.createElement("style"), r = d.querySelector(".remark-slide-scaler");
  if (!r) return;
  s.type = "text/css"; s.innerHTML = "@page {size: " + r.style.width + " " + r.style.height +"; }";
  d.head.appendChild(s);
})(document);

(function(d) {
  var el = d.getElementsByClassName("remark-slides-area");
  if (!el) return;
  var slide, slides = slideshow.getSlides(), els = el[0].children;
  for (var i = 1; i < slides.length; i++) {
    slide = slides[i];
    if (slide.properties.continued === "true" || slide.properties.count === "false") {
      els[i - 1].className += ' has-continuation';
    }
  }
  var s = d.createElement("style");
  s.type = "text/css"; s.innerHTML = "@media print { .has-continuation { display: none; } }";
  d.head.appendChild(s);
})(document);
// delete the temporary CSS (for displaying all slides initially) when the user
// starts to view slides
(function() {
  var deleted = false;
  slideshow.on('beforeShowSlide', function(slide) {
    if (deleted) return;
    var sheets = document.styleSheets, node;
    for (var i = 0; i < sheets.length; i++) {
      node = sheets[i].ownerNode;
      if (node.dataset["target"] !== "print-only") continue;
      node.parentNode.removeChild(node);
    }
    deleted = true;
  });
})();
(function() {
  "use strict"
  // Replace <script> tags in slides area to make them executable
  var scripts = document.querySelectorAll(
    '.remark-slides-area .remark-slide-container script'
  );
  if (!scripts.length) return;
  for (var i = 0; i < scripts.length; i++) {
    var s = document.createElement('script');
    var code = document.createTextNode(scripts[i].textContent);
    s.appendChild(code);
    var scriptAttrs = scripts[i].attributes;
    for (var j = 0; j < scriptAttrs.length; j++) {
      s.setAttribute(scriptAttrs[j].name, scriptAttrs[j].value);
    }
    scripts[i].parentElement.replaceChild(s, scripts[i]);
  }
})();
(function() {
  var links = document.getElementsByTagName('a');
  for (var i = 0; i < links.length; i++) {
    if (/^(https?:)?\/\//.test(links[i].getAttribute('href'))) {
      links[i].target = '_blank';
    }
  }
})();</script>
<style>
.logo {
  background-image: url(img/ncats-logo.png);
  background-size: contain;
  background-repeat: no-repeat;
  position: absolute;
  bottom: -85px;
  right: 1em;
  width: 150px;
  height: 128px;
  z-index: 0;
}
</style>

<script>
document
  .querySelectorAll(
    '.remark-slide-content' +
    ':not(.title-slide)' +
    // add additional classes to exclude here, e.g.
    // ':not(.inverse)' +
    ':not(.hide-logo)'
  )
  .forEach(el => {
    el.innerHTML += '<div class="logo"></div>';
  });
</script>

<script>
slideshow._releaseMath = function(el) {
  var i, text, code, codes = el.getElementsByTagName('code');
  for (i = 0; i < codes.length;) {
    code = codes[i];
    if (code.parentNode.tagName !== 'PRE' && code.childElementCount === 0) {
      text = code.textContent;
      if (/^\\\((.|\s)+\\\)$/.test(text) || /^\\\[(.|\s)+\\\]$/.test(text) ||
          /^\$\$(.|\s)+\$\$$/.test(text) ||
          /^\\begin\{([^}]+)\}(.|\s)+\\end\{[^}]+\}$/.test(text)) {
        code.outerHTML = code.innerHTML;  // remove <code></code>
        continue;
      }
    }
    i++;
  }
};
slideshow._releaseMath(document);
</script>
<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
(function () {
  var script = document.createElement('script');
  script.type = 'text/javascript';
  script.src  = 'https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-MML-AM_CHTML';
  if (location.protocol !== 'file:' && /^https?:/.test(script.src))
    script.src  = script.src.replace(/^https?:/, '');
  document.getElementsByTagName('head')[0].appendChild(script);
})();
</script>
  </body>
</html>
