<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="" xml:lang="">
  <head>
    <title>index.utf8.md</title>
    <meta charset="utf-8" />
    <link href="libs/remark-css-0.0.1/default.css" rel="stylesheet" />
    <link href="libs/remark-css-0.0.1/metropolis.css" rel="stylesheet" />
    <link href="libs/remark-css-0.0.1/metropolis-fonts.css" rel="stylesheet" />
  </head>
  <body>
    <textarea id="source">


class: inverse, middle
background-image: url(title_background.svg)
background-size: cover

&lt;font color = "white"&gt;
## An integrative systems genetic analysis of mammalian lipid metabolism
.pull-left[
### Andrew Patt
### Ohio State University
Department of Biomedical Informatics/Biomedical Sciences Graduate Program

]

.pull-right[
.center[
&lt;img src="img/Ohio_State_University_seal.svg.png" width="200px" /&gt;
]
]


---

&lt;style type="text/css"&gt;

p.caption {
  font-size: 0.6em;
}

.large { font-size: 200% }

.medium-large { font-size: 130% }

.small{ font-size: 80% }

.center-left {
  position:          relative;
  top:               50%;
  transform:         translateY(50%);
}
.center-right {
  position:          relative;
  top:               50%;
  transform:         translateY(10%);
}

.remark-slide-content {
  background-color: #FFFFFF;
  border-top: 80px solid #8f1414;
  font-size: 28px;
  font-weight: 300;
  line-height: 1.5;
  padding: 1em 2em 1em 2em
}

.inverse {
  background-color: #8f1414;
  text-shadow: none;
}

.left-column {
	color: #000000;
	width: 40%;
	height: 92%;
	float: left;
}

.right-column {
  width: 58%;
  float: right;
  padding-top: 1em;
}

&lt;/style&gt;


# Introduction

.pull-left[
- Hepatosteatosis (fatty liver disease) affects 30% of people in Western countries
- Often asymptomatic but can lead to fibrosis and liver cancer
- Causes are widely varied
- Genetic reference panels are like GWASs but are performed on model organisms (mice, drosophila) 
- Allows for identification of genetic, proteomic and lipidomic associations with phenotype
]

.pull-right[
&lt;img src="img/Fig1a.png" width="450px" /&gt;
]

---

# Plasma lipids correlate with some liver lipids

.left-column[
- Lipids that were significantly correlated between plasma and liver:
  - AlkylPCs
  - HexCers (MHC)
  - PEs
- These lipids have been previously reported to influence hepatic lipotoxicity
- Not all relationships were in the same lipid species or positive correlations
]

.right-column[
&lt;img src="img/Fig1c.png" width="1000px" /&gt;
]

---

# Predicting liver lipids from plasma lipids

- Objective was to predict total class abundance in liver lipids using
  pairwise ratios of individual plasma species
- Top 10% of extreme correlation coefficients were used for the model 
- Used clustering-based representative feature selection to further
  reduce number of features down to about 50 per class
- LASSO regression with five fold CV repeated 10 times for a total of 50 trials
- Predictability ranked by proportion of trials that produced a
  significant linear model (1D)

---

# Liver lipid predictions by class

.left-column[
- Points are predicted vs actual values in individual mice (n=268, top) or humans (n=58, bottom)
- Boxes are plasma lipid ratios with highest individual correlation with lipid class in liver
]

.right-column[
&lt;img src="img/Fig1e.png" width="550px" /&gt;&lt;img src="img/Fig1e-2.png" width="550px" /&gt;
]

---

# Integrating lipidome and peroxisomal proteome

- Used correlations to identify proteins associated with levels of peroxisomal "baits"
- Enrichment analysis confirmed peroxisomal proteins were recovered by this strategy
- Identified correlations between known peroxisomal proteins and liver lipid classes as well as novel associations
- ACAD11 was associated with almost every lipid class queried
- Validated it's activity in lipid metabolism in the peroxisome through:
  - Enrichment analysis of associated proteins
  - Confocal microscopy/Western blotting showing colocalization with peroxisome
  - Affinity-purification MS, which identified associations with more
    peroxisomal proteins as well as validated some correlations

---

# ACAD11 as a novel regulator of peroxisomal lipidome

.center[
&lt;img src="img/Fig2.png" width="1000px" /&gt;
]

---

# Genomic integration with proteome and lipidome

.pull-left[
- Identified 1,984 SNPs that were associated with a protein and a
lipid in the liver
-  Identified 3,342 SNPs in 27 loci that co-mapped to both a hepatic protein
and a plasma lipid, in which the protein and plasma lipid also correlated 
-  Implies causality in the identified protein/lipid relationship]

.pull-right[
&lt;img src="img/Fig3.png" width="1000px" /&gt;
Left is liver, right is plasma (top row)
]

---

# Investigating PSMD9 as a lipid metabolism regulator

- Previously known to be a component of proteasomes, associated with obesity and CVD
- Found global correlations with large number of liver and plasma lipids
- Performed gain and loss of function of PSMD9 experiments in 2 mouse
  strains being fed a Western diet (lipotoxic conditions)
- Many proteins involved in FA metabolism and synthesis were decreased following LoF
- Some effects were strain-specific
- Validated with histological analysis
---

# PSMD9 promotes lipid synthesis in the liver

.center[
&lt;img src="img/Fig5.png" width="900px" /&gt;
]

---

# Conclusions

--

.pull-left[
Conclusions drawn from multiomic correlation analysis
&lt;br&gt;
&lt;br&gt;
.center[
&lt;img src="img/spongebob.jpg" width="250px" /&gt;
]
]

--

.pull-right[
Conclusions drawn from multiomic correlation analysis **with experimental validation**

&lt;img src="img/spongebob_ripped.gif" width="500px" /&gt;
]

---

# Conclusions
.small[
- Identified lipid pairs that can be assayed in serum to predict liver
  lipid levels that are associated with hepatosteatosis (validated in humans)
- Designed a network-based approach to identifying important novel
  protein regulators of lipids in hepatosteatosis, including ACAD11 in
  the peroxisome as an example (prior knowledge/supervised approach)
- Integrated whole genome data to infer causality
- Demonstrated the importance of interplay between genome, proteome and lipidome
- Identified PSMD9 as a potential therapeutic target for reducing
  lipid accumulation in the liver
- Extensive biological validation of findings:
  - Coherent patterns across 'omes
  - Consistencies with prior knowledge (KEGG pathways/GO/Protein interactions)
  - Human data
  - Fluorescence microscopy
  - Western blotting
  - Transcriptomics
  - AP-MS
  - Histology
  ]

---

class: inverse, bottom

# My research

---

# Novel network enrichment method

.center[
&lt;img src="img/Network_project_schematic.png" width="1000px" /&gt;
]
---

# Improved enrichment method for metabolomics

&lt;img src="img/Network_figure_2.png" width="1200px" /&gt;

---

# MDM2 amplification drives aberrant lipid metabolism

### Sphingolipid metabolism
- Cross-platform shifts in HexCers, sphingomyelins
- 2 pathways from enrichment analysis
- Increase in UGCG in MDM2 high cells
- UGCG results in worse prognosis in TCGA SARC
--
### Fatty acid/pro-inflammatory lipid metabolism
- Saturated fatty acids higher in MDM2 high
- MGDG and Ac-Ser-Asp-Lys-Pro-OH depleted in MDM2 high
- Alterations in Prostaglandin/Thromboxane synthesis and Eicosanoid pathways
--
### Cholesterol metabolism
- Differential response to atorvastatin treatment
- Shifts in bile acid pathways

---

# MDM2 inhibitor actually increases MDM2 expression

&lt;img src="img/mdm2_inhibited_metabolites.png" width="1200px" /&gt;

---

# Lipidomic data validates metabolomic data 

&lt;img src="img/panel_correlation.png" width="1200px" /&gt;

---

# Future work

- Search for more transcriptomic links with MDM2 (Sphingomyelin Synthase, Fatty Acid Synthase)
- Finish writing paper
- Refine and validate network enrichment results:
  - How best to merge networks?
  - How to analyze network topology?
    </textarea>
<style data-target="print-only">@media screen {.remark-slide-container{display:block;}.remark-slide-scaler{box-shadow:none;}}</style>
<script src="https://remarkjs.com/downloads/remark-latest.min.js"></script>
<script>var slideshow = remark.create({
"highlightStyle": "github",
"countIncrementalSlides": false,
"seal": false,
"ratio": "16:9"
});
if (window.HTMLWidgets) slideshow.on('afterShowSlide', function (slide) {
  window.dispatchEvent(new Event('resize'));
});
(function(d) {
  var s = d.createElement("style"), r = d.querySelector(".remark-slide-scaler");
  if (!r) return;
  s.type = "text/css"; s.innerHTML = "@page {size: " + r.style.width + " " + r.style.height +"; }";
  d.head.appendChild(s);
})(document);

(function(d) {
  var el = d.getElementsByClassName("remark-slides-area");
  if (!el) return;
  var slide, slides = slideshow.getSlides(), els = el[0].children;
  for (var i = 1; i < slides.length; i++) {
    slide = slides[i];
    if (slide.properties.continued === "true" || slide.properties.count === "false") {
      els[i - 1].className += ' has-continuation';
    }
  }
  var s = d.createElement("style");
  s.type = "text/css"; s.innerHTML = "@media print { .has-continuation { display: none; } }";
  d.head.appendChild(s);
})(document);
// delete the temporary CSS (for displaying all slides initially) when the user
// starts to view slides
(function() {
  var deleted = false;
  slideshow.on('beforeShowSlide', function(slide) {
    if (deleted) return;
    var sheets = document.styleSheets, node;
    for (var i = 0; i < sheets.length; i++) {
      node = sheets[i].ownerNode;
      if (node.dataset["target"] !== "print-only") continue;
      node.parentNode.removeChild(node);
    }
    deleted = true;
  });
})();</script>

<script>
(function() {
  var links = document.getElementsByTagName('a');
  for (var i = 0; i < links.length; i++) {
    if (/^(https?:)?\/\//.test(links[i].getAttribute('href'))) {
      links[i].target = '_blank';
    }
  }
})();
</script>

<script>
slideshow._releaseMath = function(el) {
  var i, text, code, codes = el.getElementsByTagName('code');
  for (i = 0; i < codes.length;) {
    code = codes[i];
    if (code.parentNode.tagName !== 'PRE' && code.childElementCount === 0) {
      text = code.textContent;
      if (/^\\\((.|\s)+\\\)$/.test(text) || /^\\\[(.|\s)+\\\]$/.test(text) ||
          /^\$\$(.|\s)+\$\$$/.test(text) ||
          /^\\begin\{([^}]+)\}(.|\s)+\\end\{[^}]+\}$/.test(text)) {
        code.outerHTML = code.innerHTML;  // remove <code></code>
        continue;
      }
    }
    i++;
  }
};
slideshow._releaseMath(document);
</script>
<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
(function () {
  var script = document.createElement('script');
  script.type = 'text/javascript';
  script.src  = 'https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-MML-AM_CHTML';
  if (location.protocol !== 'file:' && /^https?:/.test(script.src))
    script.src  = script.src.replace(/^https?:/, '');
  document.getElementsByTagName('head')[0].appendChild(script);
})();
</script>
  </body>
</html>
