<!DOCTYPE html>
<html lang="" xml:lang="">
  <head>
    <title>index.utf8</title>
    <meta charset="utf-8" />
    <script src="libs/header-attrs-2.1/header-attrs.js"></script>
    <link href="libs/remark-css-0.0.1/default.css" rel="stylesheet" />
    <link href="libs/remark-css-0.0.1/metropolis.css" rel="stylesheet" />
    <link href="libs/remark-css-0.0.1/metropolis-fonts.css" rel="stylesheet" />
  </head>
  <body>
    <textarea id="source">


class: inverse, middle
background-image: url(title_background.svg)
background-size: cover

&lt;font color = "white"&gt;
## Andrew Patt Committee Meeting
.pull-left[
### May 11th, 2020
]

.pull-right[
.center[
&lt;img src="img/Ohio_State_University_seal.svg.png" width="300px" /&gt;
]
]


---

&lt;style type="text/css"&gt;

p.caption {
  font-size: 0.6em;
}

.large { font-size: 200% }

.medium-large { font-size: 130% }

.small{ font-size: 70% }

.tiny{ font-size: 55% }

.center-left {
  position:          relative;
  top:               50%;
  transform:         translateY(50%);
}
.center-right {
  position:          relative;
  top:               50%;
  transform:         translateY(10%);
}

.remark-slide-content {
  background-color: #FFFFFF;
  border-top: 80px solid #8f1414;
  font-size: 28px;
  font-weight: 300;
  line-height: 1.5;
  padding: 1em 2em 1em 2em
}

.inverse {
  background-color: #8f1414;
  text-shadow: none;
}

.right-column {
	color: #000000;
	width: 30%;
	height: 92%;
	float: right;
}

.left-column {
  width: 68%;
  float: left;
}

&lt;/style&gt;

# Agenda

--
### 1. Review of progress

--
### 2. Thesis plan

--
### 3. Plans for future work

---

# Status Update

- 4th year in Biomedical Sciences Graduate Program wrapping up in August

--

- Currently a research associate contractor at NCATS

--

- Core coursework+Biomedical informatics coursework completed

--

- Hoping to defend thesis late November/early December 2020

--

- Looking to transition to industry eventually, examining postdoc options

---
# Current Accomplishments

.small[
#### Three first author publications
- MDM2-Dependent Rewiring of Metabolomic and Lipidomic Profiles in Dedifferentiated Liposarcoma (DDLPS) Models (**MDPI Cancers**)
- Challenges in proteogenomics: a comparison of analysis methods with the case study of the DREAM proteogenomics sub-challenge (**BMC Bioinformatics**)
- Integration of Metabolomics and Transcriptomics to Identify Gene-Metabolite Relationships Specific to Phenotype (**Methods in Molecular Biology**)
]
--

.small[
#### Three middle author publications
- Metabolomics and Multi-Omics Integration: A Survey of Computational Methods and Resources (2nd author, **Metabolites**)
- RaMP: A Comprehensive Relational Database of Metabolomics Pathways for Pathway Enrichment Analysis of Genes and Metabolites. (2nd author, **Metabolites**)
- LipidMatch Flow: Automated Lipidomics Workflow shows Lipid Changes Indicative of Early Stage Alcohol Induced Liver Damage (6th author, **Submission Pending**)
]
--

.small[
#### Other Accomplishments:
- T32 departmental fellowship (2017-2019)
- Potentially fundable F31 fellowship (Score: 30)
]
---

# Timeline

.tiny[
| Relative time | Actual time | Milestone |
|-------------------|-----------------|--------------|
| Year 0 | August 2016 | Started program  |
| Year 1.5 | December 2017 | RaMP publication (2nd author)  |
| Year 2.5 | October 2018 | Candidacy exam |
| Year 2.5 | January 2019 | RaMP textbook chapter (1st author)  |
| Year 3 | June 2019 | Proteogenomic integration review (Co-1st author)  |
| Year 3.5    | April 2020    | Multiomic integration review  (2nd author) |
| Year 3.5  | May 2020    | **Liposarcoma metabolomics (Co-1st author)**   |
| Year 4 | July 2020  | **MetaboSPAN (1st author)**  |
| Year 4.5  | December 2020 | Defend and submit thesis |
| Year 4.5   | Spring 2021  | **Renovated RaMP (1st or 2nd author)**  |
| ???    | ???    | Identifying potential COVID therapies |
| ???    | ???    | LipidMatch Flow (6th author) |
]

---

# Thesis Chapters

.tiny[
**Part I: Metabolomics and Lipidomics: Applications in biomedical sciences (Introduction)**

**Part II: Metabolomics of liposarcoma**
   
   1) Liposarcoma review
   
   2) Liposarcoma treatment response and Ceramides
   
   3) Conclusions

**Part III: Computational approaches for functional analysis and integration of metabolomic and lipidomic data**

   1) Introduction
   - Overview of standard metabolomic workflows (Issues include missing annotations, identifications, and lack of standard annotations)
	  
   - Pathway analysis (Databases, methods, tools &amp; challenges)
	  
   - Integration (Methods, tools &amp; challenges)
	  
2) RaMP chapter (addressing issues in multiomic data analysis) and
      improved lipid pathway annotations
	  
3) MetaboSPAN: Network integration of metabolite annotations to
improve functional interpretation
   
4) Conclusions

**Part IV: Conclusions (Broadly state my contributions to the field of metabolomics)**
]

---
class: inverse, center, middle
# Current research

---

# Liposarcoma study objectives

--

### 1. Understand how MDM2 amplification confers resistance to statins in DDLPS in vitro using metabolomics

--

### 2. General survey of metabolome of DDLPS

--

### 3. Assess use of MDM2 inhibitors for inducing sensitivity in MDM2 higher cell lines

--

### 4. Connect metabolomic results to human transcriptomic results

---

# HexCers are upregulated by MDM2 in both panels
.pull-left[
![](index_files/figure-html/unnamed-chunk-2-1.png)&lt;!-- --&gt;
]

.pull-right[
![](index_files/figure-html/unnamed-chunk-3-1.png)&lt;!-- --&gt;
]

---

# Summary

--

### 1. MDM2 amplification was associated with resistance to apoptosis, shifts in energy metabolism, and driving a pro-inflammatory phenotype

--

### 2. Atorvastatin worked through changes in the sphingolipid pathway which were counteracted by MDM2 activity

--

### 3. MDM2 inhibition had no impact on MDM2 higher cells and affected DNA repair and amino acid metabolism in MDM2 lower cells

--

### 4. Pathway analysis was crucial for formalizing the observations drawn from our two panels
---

# F31 Aims Background
--

- Pathway annotations are not available for most metabolites identified in an experiment
--

- There's variability in the number and types of metabolites that can be identified in similar experimental conditions
--

- Lipid annotations are sparse and difficult to consolidate
--

  - Pathway coverage is poor
--

  - Alternate schemes are not harmonized
--

- ORA statistics falsely assume that pathways are independent of each other

---

# Aim 1 

### Expand and improve usability of lipid annotations within RaMP from multiple sources
--

.small[
**Subaim 1a:  Consolidate lipid annotations in RaMP from lipid databases**
]
--

.small[
- LION/Web, SwissLipids, Lipid Maps, LipidPedia, InChIKey
]
--

.small[
**Subaim 1b: Improve usability of known lipid annotations by implementing conventional ORA software
for expanded lipidomic knowledge base.**
]
--

.small[
- Structure, Chemical class, Disease, MeSH term, Biological Function,
Reaction(?), Subcellular location, Biochemical/physical properties 
]
--

.small[
**Subaim1c: Develop methodology for the visualization of coverage of pathways in user-supplied data.**
]
--

.small[
- Generate a network visual for demonstrating high level functional clusters in user analytes
]

---
# Aim 2 

### Develop a novel network-based pathway and annotation enrichment method that integrates multiple sources of metabolite/lipid annotations
--

.small[
**Subaim 2a:  Build a consensus similarity network model from multiple annotation types using sim-
ilarity network fusion algorithms**
]
--

.tiny[
- Need to determine how to quantify and weight similarities between analytes on many ontological levels
- Alternative approaches include simpler (sum edge weights, union of edges) or more complex (Grassmann manifold) strategies
]
--

.small[
**Subaim 2b: Develop methodology for detecting enriched pathways and annotations in the consen-
sus similarity network**
]
--

.tiny[
- Currently using random walks with restarts to identify crosstalkers with seed set (list of DE analytes)
- This is used to build a new subnetwork that can be tested for enrichment of annotations
- Heat diffusion is a potential alternative
]
--

.small[
**Subaim2c: Develop MetaboSPAN: R package/Shiny app for Metabolite Structure, Pathway, and Anno-
tation Network enrichment analysis**
]


---
# MetaboSPAN

&lt;img src="img/network_schematic1.png" width="1000px" /&gt;

---

# MetaboSPAN

&lt;img src="img/network_schematic2.png" width="1000px" /&gt;

---

# MetaboSPAN

&lt;img src="img/network_schematic3.png" width="1000px" /&gt;

---

# MetaboSPAN

&lt;img src="img/network_schematic4.png" width="1000px" /&gt;

---

# MetaboSPAN

&lt;img src="img/network_schematic5.png" width="1000px" /&gt;

---

# MetaboSPAN

&lt;img src="img/network_schematic6.png" width="1000px" /&gt;

---

# MetaboSPAN

&lt;img src="img/network_schematic7.png" width="1000px" /&gt;

---

# MetaboSPAN

&lt;img src="img/network_schematic8.png" width="1000px" /&gt;

---

# MetaboSPAN

&lt;img src="img/network_schematic9.png" width="1000px" /&gt;

---

# MetaboSPAN

&lt;img src="img/network_schematic10.png" width="1000px" /&gt;

---

# MetaboSPAN

&lt;img src="img/network_schematic11.png" width="1000px" /&gt;

---

# MDM2 high vs low comparison

&lt;img src="img/Network_figure_2.png" width="1000px" /&gt;

---

# Impact of MDM2 Inhibition

&lt;img src="img/MDM2_inhibitor_comparison.png" width="1000px" /&gt;

---

# MetaboSPAN moving forward

--

## What to test:
.small[
1. How much information to include (Which annotation types)
2. How to encode that information (How are edges built?)
3. How to combine network models (SNF algorithms, sum edge weights, multi-layer network)
4. How to evaluate network topology (Random walks, heat diffusion)
5. How to evaluate annotation enrichment (Fisher's vs. FSEA test)
]

--

## How to test:

.small[
- Requires both biological and statistical validation (computational efficiency is a concern as well)
- **Statistical**
  - Simulation studies showing improved detection of pathways in same list of analytes
  - Consistent results across similar experimental setups
- **Biological**
  - Consistent results across similar experimental setups
  - Results reflect known biology
  - Test generated hypotheses in *in vitro* liposarcoma experimentation
]

---
class: inverse, center, middle
# Questions?
    </textarea>
<style data-target="print-only">@media screen {.remark-slide-container{display:block;}.remark-slide-scaler{box-shadow:none;}}</style>
<script src="https://remarkjs.com/downloads/remark-latest.min.js"></script>
<script>var slideshow = remark.create({
"highlightStyle": "github",
"countIncrementalSlides": false,
"seal": false,
"ratio": "16:9"
});
if (window.HTMLWidgets) slideshow.on('afterShowSlide', function (slide) {
  window.dispatchEvent(new Event('resize'));
});
(function(d) {
  var s = d.createElement("style"), r = d.querySelector(".remark-slide-scaler");
  if (!r) return;
  s.type = "text/css"; s.innerHTML = "@page {size: " + r.style.width + " " + r.style.height +"; }";
  d.head.appendChild(s);
})(document);

(function(d) {
  var el = d.getElementsByClassName("remark-slides-area");
  if (!el) return;
  var slide, slides = slideshow.getSlides(), els = el[0].children;
  for (var i = 1; i < slides.length; i++) {
    slide = slides[i];
    if (slide.properties.continued === "true" || slide.properties.count === "false") {
      els[i - 1].className += ' has-continuation';
    }
  }
  var s = d.createElement("style");
  s.type = "text/css"; s.innerHTML = "@media print { .has-continuation { display: none; } }";
  d.head.appendChild(s);
})(document);
// delete the temporary CSS (for displaying all slides initially) when the user
// starts to view slides
(function() {
  var deleted = false;
  slideshow.on('beforeShowSlide', function(slide) {
    if (deleted) return;
    var sheets = document.styleSheets, node;
    for (var i = 0; i < sheets.length; i++) {
      node = sheets[i].ownerNode;
      if (node.dataset["target"] !== "print-only") continue;
      node.parentNode.removeChild(node);
    }
    deleted = true;
  });
})();
(function() {
  "use strict"
  // Replace <script> tags in slides area to make them executable
  var scripts = document.querySelectorAll(
    '.remark-slides-area .remark-slide-container script'
  );
  if (!scripts.length) return;
  for (var i = 0; i < scripts.length; i++) {
    var s = document.createElement('script');
    var code = document.createTextNode(scripts[i].textContent);
    s.appendChild(code);
    var scriptAttrs = scripts[i].attributes;
    for (var j = 0; j < scriptAttrs.length; j++) {
      s.setAttribute(scriptAttrs[j].name, scriptAttrs[j].value);
    }
    scripts[i].parentElement.replaceChild(s, scripts[i]);
  }
})();
(function() {
  var links = document.getElementsByTagName('a');
  for (var i = 0; i < links.length; i++) {
    if (/^(https?:)?\/\//.test(links[i].getAttribute('href'))) {
      links[i].target = '_blank';
    }
  }
})();</script>

<script>
slideshow._releaseMath = function(el) {
  var i, text, code, codes = el.getElementsByTagName('code');
  for (i = 0; i < codes.length;) {
    code = codes[i];
    if (code.parentNode.tagName !== 'PRE' && code.childElementCount === 0) {
      text = code.textContent;
      if (/^\\\((.|\s)+\\\)$/.test(text) || /^\\\[(.|\s)+\\\]$/.test(text) ||
          /^\$\$(.|\s)+\$\$$/.test(text) ||
          /^\\begin\{([^}]+)\}(.|\s)+\\end\{[^}]+\}$/.test(text)) {
        code.outerHTML = code.innerHTML;  // remove <code></code>
        continue;
      }
    }
    i++;
  }
};
slideshow._releaseMath(document);
</script>
<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
(function () {
  var script = document.createElement('script');
  script.type = 'text/javascript';
  script.src  = 'https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-MML-AM_CHTML';
  if (location.protocol !== 'file:' && /^https?:/.test(script.src))
    script.src  = script.src.replace(/^https?:/, '');
  document.getElementsByTagName('head')[0].appendChild(script);
})();
</script>
  </body>
</html>
